app/views/thredded/messageboards/index.html.erb

```diff
@@ -8,7 +8,10 @@
       <%= view_hooks.messageboards_index.list.render self, groups: @groups do %>
         <% @groups.each do |group| %>
           <% if group.name.present? %>
-            <h3 class="thredded--messageboards-group--title"><%= group.name %></h3>
+            <h3 class="thredded--messageboards-group--title">
+              <%= link_to group.name,
```

Comment:
Please add a setting to control this and use `link_to_if Thredded.show_messageboard_group_page`.

I keep track of a dozen forums using Thredded, and very few of them have more than 6 boards, so this link would be confusing.

I'm OK with the setting being `true` by default.

Reply:
Sounds good.